.drone.star

```diff
@@ -150,10 +145,9 @@ def main(ctx):
         afterCoverageTests = afterCoveragePipelines(ctx)
         dependsOn(coverageTests, afterCoverageTests)
 
-    after = afterPipelines(ctx)
-    dependsOn(afterCoverageTests + nonCoverageTests + stages, after)
+    dependsOn(afterCoverageTests + nonCoverageTests + stages)
 
```

Comment:
```suggestion
```

We need to remove the `dependOn()` call and the blank line after it, to work and pass the linter checks.